powermate.toml

```diff
@@ -1,4 +1,5 @@
 # The program checks for configuration files in the following order:
+# - $HOME/.config/powermate.toml
```

Comment:
I am wondering if it makes sense to also remove the search for `~/.powermate.toml`?

I think I've read somewhere that it should be discouraged to pollute the home directory (like I've been doing), and that programs should use `~/.config/`. But I don't remember where I read that, or if it's broadly accepted these days..

Do you happen to know where the `~/.config/` convention started and who is advocating for it?

Reply:
Yes - I searched for the documentation before making that change - https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html is the spec, and I guess that the origins of `.config/` are also within the freedesktop.org stuff.

Reply:
I'm not sure if the `~/.powermate.toml` should be removed immediately or if there should be a release where both files get read, and maybe a warning logged when the old position is used...

Reply:
I don't think this program has enough users to warrant the need for a transitory period.

Thanks for digging up that link. To do this properly, it seems like we should use the environment variable `XDG_CONFIG_HOME`, and default it to `~/.config/` if it's not set.

> $XDG_CONFIG_HOME defines the base directory relative to which user specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

Reply:
Good spot!

I've removed the stuff referring to `$HOME` and replaced it by using `$XDG_CONFIG_HOME` and `~/.config/`.